python/practices/practice-3/src/path1/required_letter_word_counter.py

```diff
@@ -0,0 +1,61 @@
+"""
+Takes in a list of words and a string of characters, checks and counts the
+words using all the characters in the string and prints the result.
+"""
+
+
+import os
+
+
+def uses_all(word, required_letters):
+    """
+    Check if a word contains all the required letters.
+
+    Parameters:
+        - word (str): The word to check.
+        - required_letters (str): A string containing the letters that must be
+        present in 'word'.
+
+    Returns:
+        - bool: True if 'word' contains all the required letters, False
+        otherwise.
+    """
+
+    for letter in required_letters:
+        if letter not in word:
+            return False
+
+    return True
+
+
+def count_words(words, required_letters):
+    """
+    Count the number of words in a list that contain all the required letters.
+
+   Parameters:
```

Comment:
Pay attention to the distance from the left margin.

Reply:
Done